### PR TITLE
tasks from HM9

### PR DESCRIPTION
**Pull request by MichaelN98**

### CODE REVIEW

File: HM9/HM9_task1.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
+class Any_company:
```
Comment (bohdandovbsyh) [inline HM9/HM9_task1.py]: in the class definition, we use CamleCase style.
AnyCompany

File: HM9/HM9_task1.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    """A class that describes a company"""
+
+    def __init__(self, name, age, budget):
```
Comment (bohdandovbsyh) [inline HM9/HM9_task1.py]: please add annotations

File: HM9/HM9_task1.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    def name(self, name):
+        """
+        Set the name of the company
+
+         Parameters
+         ----------
+         name : str
+         The name of the company with that conditions
+        """
+        if len(name) >= 3:
```
Comment (bohdandovbsyh) [inline HM9/HM9_task1.py]: looks like I can set list, dict, tuple

File: HM9/HM9_task1.py [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+    @property
+    def  budget(self):
+        """
+        Get the  budget of the company
+        """
+        return self.__budget
+
+    @ budget.setter
+    def  startup(self,  budget):
```
Comment (bohdandovbsyh) [inline HM9/HM9_task1.py]: why do you have another name?

File: HM9/HM9_task1.py [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+         The  budget of the company with that conditions
+        """
+        if not isinstance( budget, int):
+            raise ValueError(" budget must be an integer")
+        if  budget <= 0:
+            raise ValueError(" budget cannot be negative")
+        if  budget < 10000:
+            print(f'{ budget} is a startup')
+        if  budget >= 10000:
+            print(f'{ budget} is a business')
```
Comment (bohdandovbsyh) [inline HM9/HM9_task1.py]: where do you set value?

File: HM9/HM9_task1.py [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+    @staticmethod
+    def bonus_of_timework(years_of_work):
+        """
+        Add bonus for work in the company
+        Parameters
+         ----------
+         bonus : int
+        """
+        return years_of_work * 0.5
```
Comment (bohdandovbsyh) [inline HM9/HM9_task1.py:59]: also please add some general functions for this class

File: HM9/HM9_task2.py [hunk 7]
```diff
@@ -0,0 +1,109 @@
+class Worker:
+    """A class that describes a worker"""
+
+    def __init__(self, name, age, salary):
```
Comment (bohdandovbsyh) [inline HM9/HM9_task2.py]: please add annotations

File: HM9/HM9_task2.py [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+    def name(self, name):
+        """
+        Set the name of the company
+
+         Parameters
+         ----------
+         name : str
+         The name of the company with that conditions
+        """
+        if len(name) >= 2:
```
Comment (bohdandovbsyh) [inline HM9/HM9_task2.py]: it can be list dict ..

File: HM9/HM9_task1.py [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+          budget : int
+         The  budget of the company with that conditions
+        """
+        if not isinstance( budget, int):
+            raise ValueError(" budget must be an integer")
+        if  budget <= 0:
+            raise ValueError(" budget cannot be negative")
+        if  budget < 10000:
+            print(f'{ budget} is a startup')
+        if  budget >= 10000:
```
Comment (bohdandovbsyh) [inline HM9/HM9_task1.py]: use elif . 
not if every time

File: HM9/HM9_task2.py [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+         ----------
+        salary : int
+         The the salary of the workers with that conditions
+        """
+        if salary <= 1000:
+            print(f'{salary} is small salary')
+        if salary >= 1000 and salary <= 2000:
+            print(f'{salary} is good salary')
+        if salary > 2000 :
+            print(f'{salary} is big salary')
```
Comment (bohdandovbsyh) [inline HM9/HM9_task2.py:59]: where do you set this value?

File: HM9/HM9_task2.py [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+    @staticmethod
+    def time_work(years_of_work):
+        """
+        Add bonus for work in the company if worked more 8 years
+        Parameters
+         ----------
+         years_of_work : int
+        """
+        if years_of_work > 8:
+            return years_of_work * 0.5
```
Comment (bohdandovbsyh) [inline HM9/HM9_task2.py:66]: please add some general functions as well

File: HM9/HM9_task1.py (hunk 1 above)
Comment (MichaelN98) [reply HM9/HM9_task1.py]: fix

File: HM9/HM9_task1.py (hunk 2 above)
Comment (MichaelN98) [reply HM9/HM9_task1.py]: fix

File: HM9/HM9_task1.py (hunk 3 above)
Comment (MichaelN98) [reply HM9/HM9_task1.py]: @bohdandovbsyh  I  don`t understand?

File: HM9/HM9_task1.py (hunk 4 above)
Comment (MichaelN98) [reply HM9/HM9_task1.py]: fix

File: HM9/HM9_task1.py (hunk 9 above)
Comment (MichaelN98) [reply HM9/HM9_task1.py]: fix

File: HM9/HM9_task1.py (hunk 5 above)
Comment (MichaelN98) [reply HM9/HM9_task1.py]: @bohdandovbsyh 
I need to set a value in the property?

File: HM9/HM9_task2.py (hunk 7 above)
Comment (MichaelN98) [reply HM9/HM9_task2.py]: fix

File: HM9/HM9_task1.py (hunk 3 above)
Comment (MichaelN98) [reply HM9/HM9_task1.py]: add

File: HM9/HM9_task2.py (hunk 11 above)
Comment (MichaelN98) [reply HM9/HM9_task2.py:66]: add

File: HM9/HM9_task2.py (hunk 8 above)
Comment (MichaelN98) [reply HM9/HM9_task2.py]: fix

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**